python/lsst/analysis/tools/actions/vector/vectorActions.py

```diff
@@ -144,9 +145,9 @@ def getInputSchema(self) -> KeyedDataSchema:
         return ((self.vectorKey, Vector),)
 
     def __call__(self, data: KeyedData, **kwargs) -> Vector:
-        with np.warnings.catch_warnings():  # type: ignore
-            np.warnings.filterwarnings("ignore", r"invalid value encountered")  # type: ignore
-            np.warnings.filterwarnings("ignore", r"divide by zero")  # type: ignore
+        with warnings.catch_warnings():
```

Comment:
@natelust it looks like the mypy configuration does not warn about irrelevant ignores.